android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/NearbyBeaconsFragment.java

```diff
@@ -389,7 +389,47 @@ private void safeNotifyChange() {
     }
 
     public void addItem(PwoMetadata pwoMetadata) {
-      mPwoMetadataList.add(pwoMetadata);
+      // If multiple PWOs have the same domain and page title, we consider them to be in the same
```

Comment:
..and instead of describing the specific set of actions, consider describing the reason we are doing this:

"Multiple urls from the same group basically represent the same result, but with minimally different URLs (e.g. like a differing hash variable).  We would like to not display them all and confuse the user -- instead, we display one grouped result, and take the user to URL stored on the closest beacon when they click"